src/javafundamentals/collections/list/Ex02Iterate.java

```diff
@@ -1,9 +1,27 @@
 package javafundamentals.collections.list;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Create a List with numbers from 1 to 10.
  * Print the sum of all the values in the list.
  */
 
 public class Ex02Iterate {
+    public static void main(String[] args) {
+        List<Integer> integerList = new ArrayList<>();
+
+        for (int i = 0; i <= 10; i++) {
+            integerList.add(i);
+        }
+
+        int total = 0;
+        for (int i:
```

Comment:
Just small detail about formatting, i notice that you always format the foreach loops as this one, the standard way is:
       ```
 for (int i: integerList) {
              total += i;
        }
```

Reply:
Ah okay, intelliJ always puts the bracket on the second line down so I thought i would copy that. Good to know!